packages/core/plugins/core-plugins-server-internal/src/plugins_service.test.ts

```diff
@@ -721,12 +721,7 @@ describe('PluginsService', () => {
         config: {
           additionalPluginPaths: [],
           initialize: true,
-          pluginSearchPaths: [
-            resolve(process.cwd(), 'src', 'plugins'),
-            resolve(process.cwd(), 'x-pack', 'plugins'),
-            resolve(process.cwd(), 'plugins'),
-            resolve(process.cwd(), '..', 'kibana-extra'),
-          ],
```

Comment:
Why was `resolve(process.cwd(), 'plugins'),` removed? Shouldn't we keep it as we do for `kibana-extra`?

Reply:
Hmm, I guess I should probably test installing a third party plugin... I think you're right

Reply:
Fixed in [`8ecb115` (#148130)](https://github.com/elastic/kibana/pull/148130/commits/8ecb115e714d7067b6d0880e3aee657a1d440d13)

Reply:
successfully validated that this doesn't cause any issues with third-party plugins. Generated, built, and installed into a build successfully.